pythonie/core/static/css/bootstrap.css

```diff
@@ -5046,7 +5046,7 @@ a.badge:focus {
 }
 .thumbnail {
   display: block;
-  padding: 4px;
+  padding: 4px 0 4px 0;
```

Comment:
Can we add this as an exception in the site.css instead of bootstrap. This change would be lost if we ever upgrade bootstrap in the future.